developer_manual/digging_deeper/continuous_integration.rst

```diff
@@ -113,37 +99,19 @@ Static analysis
         </issueHandlers>
     </psalm>
 
-.. Note:: The definition suppresses usages of the global and static class ``OC`` like ``\OC::$server``, which is discouraged but still found in some apps. The doctrine supression is currently necessary as the database mappers and schema abstractions leak some of the 3rd party libraries of Nextcloud that are not known to Psalm.
+.. Note:: The definition suppresses usages of the global and static class ``OC`` like ``\OC::$server``, which is
+discouraged but still found in some apps. The doctrine suppression is currently necessary as the database mappers and
+schema abstractions leak some of the 3rd party libraries of Nextcloud that are not known to Psalm.
 
 
 You can put this process into a GitHub Action that is run for every pull request.
-
-.. code-block:: yaml
-
-    name: Static analysis
-    on: [push]
-    jobs:
-    static-psalm-analysis:
-        runs-on: ubuntu-latest
-        strategy:
-            matrix:
-                ocp-version: [ 'dev-master', 'v20.0.0' ]
-        name: Nextcloud ${{ matrix.ocp-version }}
-        steps:
-            - name: Checkout
-                uses: actions/checkout@master
-            - name: Set up php
-                uses: shivammathur/setup-php@master
-                with:
-                    php-version: 7.4
-                    coverage: none
-            - name: Install dependencies
-                run: composer i
-            - name: Install dependencies
-                run: composer require --dev christophwurst/nextcloud:${{ matrix.ocp-version }}
-            - name: Run coding standards check
-                run: composer run psalm
-
-This creates a matrix, where the app is tested against ``dev-master``, the latest version of ``OCP`` found in the main branch of Nextcloud server, as well as ``v20.0.0``, the currently latest stable release. Adjust this to your needs.
+Check our `simple github action <https://github.com/nextcloud/.github/blob/master/workflow-templates/psalm.yml>`_ from
+our `nextcloud template repository <https://github.com/nextcloud/.github>`_.
+
+If you want to support multiple versions of Nextcloud server with a single app version, checkout this slightly
+`more complex action <https://github.com/nextcloud/.github/blob/master/workflow-templates/psalm-matrix.yml>`_.
+This creates a matrix, where the app is tested against ``dev-master``, the latest version of ``OCP`` found in the main
```

Comment:
Maybe formulate this more generic because I'm afraid we might forget to update this info here